### PR TITLE
[ACA-4690] added width and max-width to sidebar

### PR DESCRIPTION
**Pull request by arohilaGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

The width of the Create tag keeps increasing indefinitely.

**What is the new behaviour?**

Limiting the width.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss [hunk 1]
```diff
@@ -56,6 +56,14 @@
     }
   }
 
+  .sidebar {
```
Comment (AleksanderSklorz) [inline projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss:59]: I'm wondering if that is correct place for these styles. In html for this component I can't see that this class is used. Also note that this component has encapsulation: ViewEncapsulation.None, so having this class not nested in scss may cause that you affect other places with sidebar class too. @MichalKinas what do you think?

Comment (MichalKinas) [reply projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss:59]: Yeah I agree, component itself shouldn't style elements that it's not aware of. In this case element containing sidebar class comes from ng-content

Comment (arohilaGL) [reply projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss:59]: This was removed in a previous commit, I only identified what got removed and placed it back.

**Review by MichalKinas — APPROVED** [review]

**Review by AleksanderSklorz — APPROVED** [review]

File: projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss (hunk 1 above)
Comment (AleksanderSklorz) [reply projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss:59]: discussed on slack, please leave and for now and raise jira as suggested by @MichalKinas

**Pull request merged**